pynecone/components/datadisplay/table.py

```diff
@@ -143,7 +143,13 @@ def create(cls, *children, cell_type: str = "", cells=None, **props) -> Componen
         types = {"header": Th, "data": Td}
         cell_cls = types.get(cell_type)
         if len(children) == 0 and cell_cls:
-            children = [cell_cls.create(cell) for cell in cells or []]
+            from pynecone.components.layout.foreach import Foreach
+            from pynecone.var import Var
```

Comment:
Can we move these imports to the top of the file?

Reply:
Done.